### PR TITLE
[Customers Product]: Change references from 'Users' to 'Customers' within documentation 👥 

### PR DESCRIPTION
**Pull request by sumitramanga**
## [Customers Product]: Change references from 'User tracking' to 'Customers' 👤

**NOTE: This is not to be merged until we have fully released the name change in the Raygun application. Changes to 'users' within the code need to be discussed and planned for.**

### Description 📝 
This PR is to updating the name convention for 'User tracking' to now be 'Customers' to reflect what's within the Raygun application.

**Updates**
👉 Update `README` documentation

### Test plan 🧪 
- Make sure documentation changes are reflected to be 'Customers' instead of 'Users' or 'User tracking' for the users product

### Checklist ✔️ 
- [ ] Builds pass
- [ ] Reviewed by another developer
- [ ] Code is written to standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
...
-Below are the various properties that you can use to describe the user. The Identifier is the only required field, which can be provided through the constructor.
+If a single string is not enough to describe the information that you want to log about a customer, then you can set the `UserInfo` property.
+Below are the various properties that you can use to describe the customer. The Identifier is the only required field, which can be provided through the constructor.
 
-**Identifier** The unique identifier you want to use to identify this user. Suggestions for what you could set this to are listed in the User property section above.
+**Identifier** The unique identifier you want to use to identify this customer. Suggestions for what you could set this to are listed in the User property section above.
 
-**IsAnonymous** A flag indicating whether the user is logged in (or identifiable) or if they are anonymous. An anonymous user still requires an identifier.
+**IsAnonymous** A flag indicating whether the customer is logged in (or identifiable) or if they are anonymous. An anonymous user/customer still requires an identifier.
```
Comment (QuantumNightmare) [inline README.md]: Was it intentional to leave a mention of "user" here?

File: README.md [hunk 2]
```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
 Common Features
```
Comment (QuantumNightmare) [inline README.md:303]: Way at the top of this file is a table of contents that you'll need to update. User tracking needs to be changed to Customers (both what's displayed, and the anchor to link down to this section which doesn't currently work due to the name change down here).

File: README.md [hunk 3]
```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
...
-**Identifier** The unique identifier you want to use to identify this user. Suggestions for what you could set this to are listed in the User property section above.
+**Identifier** The unique identifier you want to use to identify this customer. Suggestions for what you could set this to are listed in the User property section above.
 
-**IsAnonymous** A flag indicating whether the user is logged in (or identifiable) or if they are anonymous. An anonymous user still requires an identifier.
+**IsAnonymous** A flag indicating whether the customer is logged in (or identifiable) or if they are anonymous. An anonymous user/customer still requires an identifier.
 
-**UUID** A device identifier. Could be used to identify users across devices, or machines that are breaking for many users.
+**UUID** A device identifier. Could be used to identify customers across devices, or machines that are breaking for many customers.
 
 **Email**, **FullName** and **FirstName** are self explanatory.
```
Comment (QuantumNightmare) [inline README.md:338]: The RUM behaviour section below may also need to have some word changes

File: README.md [hunk 4]
```diff
@@ -303,36 +303,36 @@ If there currently isn't an active session, then calling this method does nothin
 Common Features
 ===============
```
Comment (QuantumNightmare) [inline README.md:305]: This repository also has a readme.txt file with duplicate content to this file that you may also want to update. That txt file is what's displayed in an IDE after installing the NuGet package.

**Review by QuantumNightmare — CHANGES_REQUESTED** [review]
Main request is regarding the table of contents because the anchor link is now broken. Added a few other comment to point out places that may have been missed.

File: README.md (hunk 1 above)
Comment (sumitramanga) [reply README.md]: Yes, but we can remove this now.  Wasn't too sure to associate anonymous user with customers or just users. But we are showing anonymous users within Customers, therefore, would make sense to be consistent with customer

File: README.md (hunk 2 above)
Comment (sumitramanga) [reply README.md:303]: Good catch, changes this

File: README.md (hunk 3 above)
Comment (sumitramanga) [reply README.md:338]: Edited this

File: README.md (hunk 4 above)
Comment (sumitramanga) [reply README.md:305]: Have edited the .txt file to reflect what's within the README

File: README.md (hunk 2 above)
Comment (QuantumNightmare) [reply README.md:303]: I would suggest also changing "User Tracking" to "Customers" to match the heading of that section.

File: readme.txt [hunk 5]
```diff
@@ -284,7 +284,7 @@ If there currently isn't an active session, then calling this method does nothin
 COMMON FEATURES
 ========================================================================================================================================================
 
-User tracking
+Customers
```
Comment (QuantumNightmare) [inline readme.txt:287]: There is still a lot of mention of the word "user" in this Customers section compared to the .md file. Might have been missed?

**Review by QuantumNightmare — CHANGES_REQUESTED** [review]
Might as well make the name in the README.md table of contents consistent. Other comment is optional.

File: readme.txt (hunk 5 above)
Comment (sumitramanga) [reply readme.txt:287]: Haved changed this out. Lmk how this looks now :)

File: README.md (hunk 2 above)
Comment (sumitramanga) [reply README.md:303]: Ah yes, have changed this.

**Review by QuantumNightmare — APPROVED** [review]
Thanks :+1:

**Pull request merged**